lexicons/com/atproto/sync/subscribeAllRepos.json

```diff
@@ -46,6 +50,16 @@
       "errors": [
         {"name": "FutureCursor"}
       ]
+    },
+    "repoOp": {
+      "type": "object",
+      "required": ["action", "path", "cid"],
+      "nullable": ["cid"],
+      "properties": {
+        "action": {"type": "string"},
```

Comment:
I wonder if there could be some value in tacking `knownValues` on here.  Since the actions are closed (create, update, delete) I don't think that tokens would be needed.

Reply:
yup that's a good point 👍